packages/reports/addon/services/report-action-dispatcher.js

```diff
@@ -1,20 +1,23 @@
 /**
- * Copyright 2017, Yahoo Holdings Inc.
+ * Copyright 2018, Yahoo Holdings Inc.
  * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
  */
 import RequestActionDispatcher from './request-action-dispatcher';
 import { RequestActions } from './request-action-dispatcher';
 
 export const ReportActions = Object.assign({} , RequestActions, {
   UPDATE_TABLE_COLUMN_ORDER: 'updateColumnOrder',
-  UPDATE_TABLE_COLUMN: 'updateColumn'
+  UPDATE_TABLE_COLUMN: 'updateColumn',
+  DELETE_REPORT: 'deleteReport'
 });
 
 export default RequestActionDispatcher.extend({
+
   /**
    * @property {Array} consumers - concatenate consumers in request
    */
   consumers: [
-    'report/table-visualization'
+    'report/table-visualization',
+    'report/report'
```

Comment:
Don't these two consumers have different contracts?

https://github.com/yahoo/navi/blob/master/packages/reports/addon/consumers/report/table-visualization.js#L22

Also, the `RequestActionDispatcher` inherits all  `RequestActionDispatcher` consumers.